StortSpelProjekt/Game/src/main.cpp

```diff
@@ -52,11 +53,10 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE, LPSTR, int nCmdShow)
     Scene* demoScene = GameScene(sceneManager);
     Scene* shopScene = ShopScene(sceneManager);
     Scene* gameOverScene = GameOverScene(sceneManager);
+    Scene* mainMenuScene = MainMenuHandler::GetInstance().CreateScene(sceneManager);
 
-    //Scene* shopScene = ShopScene(sceneManager);
-    sceneManager->SetScene(demoScene);
+    sceneManager->SetScenes(mainMenuScene);
```

Comment:
Its not called SetScenes any more. its called SetScene. This gave me compilation error.

Reply:
Fixed